packages/sdk/src/controllers/VariableController.ts

```diff
@@ -352,6 +352,33 @@ export class VariableController {
         return this.setPlaceholderInternal(id, null);
     };
 
+    /**
+     * Internal private method to set/reset a help text for a variable
+     */
+    private setHelpTextInternal = async (id: string, helpText: string | null) => {
```

Comment:
I don't like the word internal, either use # prefix (#setHelpText) or use a different name